indico/modules/vc/views.py

```diff
@@ -26,6 +26,7 @@ class WPVCJinjaMixin(WPJinjaMixin):
 
 
 class WPVCManageEvent(WPVCJinjaMixin, WPConferenceModifBase):
+    sidemenu_option = 'vc'
```

Comment:
`videoconference`? I kind of like `vc` but at least the conference menu entry already uses `videoconference` in its name